compiler/rustc_middle/src/mir/coverage.rs

```diff
@@ -51,6 +51,15 @@ rustc_index::newtype_index! {
     pub struct ExpressionId {}
 }
 
+rustc_index::newtype_index! {
+    /// ID of a mcdc decision. Used to identify decision in a function.
+    #[derive(HashStable)]
+    #[encodable]
+    #[orderable]
+    #[debug_format = "DecisionId({})"]
+    pub struct DecisionId {}
+}
```

Comment:
Currently this ID type seems to only be used in `rustc_mir_build`.

Can it be moved into that crate and made private, instead of being here? Or are you planning to use it from multiple crates later on?

Reply:
Sure, moved at [c42a016](https://github.com/rust-lang/rust/pull/126677/commits/c42a016724322166cb6e05b7391553fe58e11e89)